sdk/typescript/src/signers/txn-data-serializers/txn-data-serializer.ts

```diff
@@ -139,6 +164,18 @@ export type UnserializedSignableTransaction =
   | {
       kind: 'publish';
       data: PublishTransaction;
+    }
+  | {
+      kind: 'requestAddDelegation';
```

Comment:
I actually don't think we should add these to singable transactions or at the top provider level. We can just expose functions that generate the corresponding move call (which is directionally where we want to go with transaction builder classes).